contracts/mocks/ERC165Mock.sol

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
I think this entire file can be deleted.

Reply:
But then `ERC165Storage` should not be abstract, right ?